docs/content/latest/api/ysql/commands/ddl_alter_table.md

```diff
@@ -100,15 +100,40 @@ If enabled, row level security policies will be applied when the user is the tab
 If disabled (the default) then row level security will not be applied when the user is the table owner.
 See [CREATE POLICY](../dcl_create_policy) for details on how to create row level security policies.
 
-### *alter_table_constraint*
+### Constraints
 
-Specify a table constraint.
+Specify a table or column constraint.
 
 #### CONSTRAINT *constraint_name*
 
 Specify the name of the constraint.
 
-#### CHECK ( expression ) | FOREIGN KEY ( column_names ) *reference_clause*
+#### Foreign key
+
+`FOREIGN KEY` and `REFERENCES` specify that the set of columns can only contain values that are present in the referenced columns of the referenced table. It is used to enforce referential integrity of data.
+
+#### Unique
+
+This enforces that the set of columns specified in the `UNIQUE` constraint are unique in the table, that is, no two rows can have the same values for the set of columns specified in the `UNIQUE` constraint.
+
+#### Check
+
+This is used to enforce that data in the specified table meets the requirements specified in the `CHECK` clause.
+
+#### Default
+
+This is used to specify a default value for the column. If an `INSERT` statement does not specify a value for the column, then the default value is used. If no default is specified for a column, then the default is NULL.
+
+#### Deferrable constraints
+
+Constraints can be deferred using the `DEFERRABLE` clause. Currently, only foreign key constraints
+can be deferred in YugabyteDB. A constraint that is not deferrable will be checked at the end of every
+statement. In the case of deferrable constraints, the checking of the constraint can be postponed
```

Comment:
nit: technically it could be after each row within a statement.

Reply:
Done.